app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/EventBadge.kt

```diff
@@ -37,6 +37,7 @@ fun EventBadgeFactory(eventType: BadgeEventType, modifier: Modifier = Modifier)
         BadgeEventType.Deleted -> DeletedLabel(modifier)
         // TODO BadgeEventType.Knock -> KnockBadge(modifier)
         BadgeEventType.None -> {}
+        else -> {}
```

Comment:
are the else needed here? 
i would rather be like to explicitly list the remaining options, just to avoid errors in the future.

Reply:
makes sense, I will update that